### PR TITLE
(SHIP-1481) Increase default LimitDefault for CCIP

### PR DESCRIPTION
**Pull request by HelloKashif**
LimitDefault of 8M is the most common value used in chains so setting this as the default

### CODE REVIEW

File: core/chains/evm/config/toml/defaults/fallback.toml [hunk 1]
```diff
@@ -32,7 +32,7 @@ Mode = 'BlockHistory'
 PriceDefault = '20 gwei'
 PriceMax = '115792089237316195423570985008687907853269984665.640564039457584007913129639935 tether'
 PriceMin = '1 gwei'
-LimitDefault = 500_000
+LimitDefault = 8_000_000
```
Comment (RensR) [inline core/chains/evm/config/toml/defaults/fallback.toml:39]: Let's use this nice underscore_format_in other places as well :)

Comment (RensR) [reply core/chains/evm/config/toml/defaults/fallback.toml:39]: I mean for the 8m values you put everywhere, not unrelated entires in the fallback. Please undo any unrelated changes

File: testdata/scripts/node/db/help.txtar
Comment (jmank88) [inline testdata/scripts/node/db/help.txtar:1]: IIRC these need to be exact matches

File: core/config/docs/chains-evm.toml [hunk 2]
```diff
@@ -160,9 +160,9 @@ PriceMax = '115792089237316195423570985008687907853269984665.6405640394575840079
 PriceMin = '1 gwei' # Default
 # LimitDefault sets default gas limit for outgoing transactions. This should not need to be changed in most cases.
 # Some job types, such as Keeper jobs, might set their own gas limit unrelated to this value.
-LimitDefault = 500_000 # Default
+LimitDefault = 8000000 # Default
```
Comment (jmank88) [inline core/config/docs/chains-evm.toml]: Let's retain the format for readability:
```suggestion
LimitDefault = 8_000_000 # Default
```

**Comment by RensR** [comment]
Please remove the changesets as we no longer use them. Also please us the `8_000_000` format over `8000000`

File: .changeset/README.md
Comment (jmank88) [inline .changeset/README.md:1]: Why are all of these changeset files being deleted?

File: core/chains/evm/config/config_test.go [hunk 3]
```diff
@@ -232,8 +232,8 @@ func TestChainScopedConfig_GasEstimator(t *testing.T) {
 	assert.Equal(t, assets.GWei(20), ge.PriceDefault())
 	assert.Equal(t, assets.GWei(500), ge.PriceMax())
 	assert.Equal(t, assets.GWei(1), ge.PriceMin())
-	assert.Equal(t, uint64(500000), ge.LimitDefault())
-	assert.Equal(t, uint64(500000), ge.LimitMax())
+	assert.Equal(t, uint64(8000000), ge.LimitDefault())
+	assert.Equal(t, uint64(8000000), ge.LimitMax())
```
Comment (jmank88) [inline core/chains/evm/config/config_test.go]: Go supports the `_` format too:
```suggestion
	assert.Equal(t, uint64(8_000_000), ge.LimitDefault())
	assert.Equal(t, uint64(8_000_000), ge.LimitMax())
```

File: .changeset/README.md
Comment (simsonraj) [reply .changeset/README.md:1]: +1

File: .github/workflows/integration-tests.yml [hunk 4]
```diff
@@ -535,6 +535,7 @@ jobs:
         env:
           BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }}
           TEST_BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }}
+          DEBUG_RESTY: true
```
Comment (simsonraj) [inline .github/workflows/integration-tests.yml]: do we need to let this on?

File: core/chains/evm/config/toml/defaults/fallback.toml [hunk 5]
```diff
@@ -5,9 +5,9 @@ FinalityDepth = 50
 FinalityTagEnabled = false
 LogBackfillBatchSize = 1000
 LogPollInterval = '15s'
-LogKeepBlocksDepth = 100000
+LogKeepBlocksDepth = 100_000
 # CCIP uses paging when removing logs to avoid pushing too much pressure on the database
-LogPrunePageSize = 10000
+LogPrunePageSize = 10_000
```
Comment (simsonraj) [inline core/chains/evm/config/toml/defaults/fallback.toml]: I would keep this PR clean only for the changes related to the problem & not touch these fields for now

File: core/chains/evm/config/toml/defaults/fallback.toml [hunk 6]
```diff
@@ -79,4 +79,4 @@ DeltaCJitterOverride = '1h'
 ObservationGracePeriod = '1s'
 
 [OCR2.Automation]
-GasLimit = 5400000
+GasLimit = 5_400_000
```
Comment (simsonraj) [inline core/chains/evm/config/toml/defaults/fallback.toml]: Same here, please dont change the formats for other values as part of this PR

File: core/internal/features/features_test.go [hunk 7]
```diff
@@ -385,7 +385,7 @@ func TestIntegration_DirectRequest(t *testing.T) {
 			// Fund node account with ETH.
 			n, err := b.NonceAt(testutils.Context(t), operatorContracts.user.From, nil)
 			require.NoError(t, err)
-			tx = cltest.NewLegacyTransaction(n, sendingKeys[0].Address, assets.Ether(100).ToInt(), 21000, big.NewInt(1000000000), nil)
+			tx = cltest.NewLegacyTransaction(n, sendingKeys[0].Address, assets.Ether(801).ToInt(), 21000, big.NewInt(1000000000), nil)
```
Comment (simsonraj) [inline core/internal/features/features_test.go]: odd value 801?

File: integration-tests/testconfig/default.toml [hunk 8]
```diff
@@ -12,7 +12,7 @@ image = "public.ecr.aws/chainlink/chainlink"
 version = "2.12.0"
 
 [Common]
-chainlink_node_funding = 0.5
+chainlink_node_funding = 1
```
Comment (simsonraj) [inline integration-tests/testconfig/default.toml]: Is this enough to cover the tests?

**Review by simsonraj — CHANGES_REQUESTED** [review]

File: .changeset/README.md
Comment (amaechiokolobi) [reply .changeset/README.md:1]: @RensR mentioned earlier to remove them as they are no longer being used https://github.com/smartcontractkit/ccip/pull/664#issuecomment-2178238991

File: integration-tests/testconfig/default.toml (hunk 8 above)
Comment (amaechiokolobi) [reply integration-tests/testconfig/default.toml]: It seems to be fine for the tests, but I will increase it to 8 to align with the gas limit changes. I had this set as I previously had issues with the smoke tests before but now that its fixed it should be fine

File: core/internal/features/features_test.go (hunk 7 above)
Comment (amaechiokolobi) [reply core/internal/features/features_test.go]: The test required 800 eth + ~1 gwei, I will change this to 1000 so it is cleaner

File: integration-tests/testconfig/default.toml (hunk 8 above)
Comment (amaechiokolobi) [reply integration-tests/testconfig/default.toml]: Set this to 2, as I was getting many errors when it was set to 4

**Review by simsonraj — APPROVED** [review]

File: .changeset/README.md
Comment (RensR) [reply .changeset/README.md:1]: I meant remove the ones you created, not every single one that exists

Comment (amaechiokolobi) [reply .changeset/README.md:1]: my mistake I misunderstood the message 🤦 I'll add these back now

Comment (amaechiokolobi) [reply .changeset/README.md:1]: the changesets were re-added when merging, so this issue has been corrected @RensR

File: .github/workflows/integration-tests.yml [hunk 9]
```diff
@@ -584,6 +584,7 @@ jobs:
         env:
           BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }}
           TEST_BASE64_CCIP_CONFIG_OVERRIDE: ${{ steps.set_override_config.outputs.base_64_override }}
+          DEBUG_RESTY: false
```
Comment (mateusz-sekara) [inline .github/workflows/integration-tests.yml:592]: Why do we need this change?

**Review by mateusz-sekara — APPROVED** [review]
Looks good, one minor question about change in the CI files

File: .github/workflows/integration-tests.yml (hunk 9 above)
Comment (amaechiokolobi) [reply .github/workflows/integration-tests.yml:592]: @mateusz-sekara We only had it enabled to debug a prior issue. Or do you think I should remove the line entirely?

**Review by jasonmci — APPROVED** [review]
Approved.

**Pull request merged**